src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_RootScreen.tsx

```diff
@@ -46,14 +46,15 @@ import {ContentHeading} from '@atb/components/heading';
 import {FullScreenView} from '@atb/components/screen-view';
 import {TransitionPresets} from '@react-navigation/stack';
 import {formatPhoneNumber} from '@atb/utils/phone-number-utils.ts';
+import { useIsTravelAidEnabled } from '@atb/travel-aid/use-is-travel-aid-enabled';
 
 const buildNumber = getBuildNumber();
 const version = getVersion();
 
 type ProfileProps = ProfileScreenProps<'Profile_RootScreen'>;
 
 export const Profile_RootScreen = ({navigation}: ProfileProps) => {
-  const {enable_ticketing} = useRemoteConfig();
+  const {enable_ticketing, enable_vipps_login} = useRemoteConfig();
```

Comment:
We do nothing with `enable_vipps_login` it was just moved here. Before there were two `useRemoteConfig` usages for different atrrs.